.github/workflows/deploy-to-wordpress.yml

```diff
@@ -0,0 +1,91 @@
+name: Deploy to WordPress.org
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
```

Comment:
🤔 Do you want to deploy whenever a commit is merged to the main branch?

Looking at the rest of the file, you want to do that when creating a new tag/release.

Reply:
Alley has Github actions to create automatic tag releases. You might want to use them and leave this only to deploy the plugin to WordPress.